Providers/Scripts/2.4x-2.5x/Scripts/Tests/test_nxProviders.py

```diff
@@ -3155,88 +3155,88 @@ def testGetOMSAgent_del(self):
-##        d = { 'Name': 'SimpleCustomLog', 'EnableCustomLogConfiguration': True, 'CustomLogObjects': [{ 'LogName': 'LinuxSampleCustomLog1', 'FilePath': [ '/tmp/test1.log', '/tmp/logs/*.log' ] }, { 'LogName': 'LinuxSampleCustomLog2', 'FilePath': [ '/tmp/test2.log' ] } ] }
-##        for customlog in d['CustomLogObjects']:
-##            customlog['LogName'] = nxOMSCustomLog.protocol.MI_String(customlog['LogName'])
-##            customlog['FilePath'] = nxOMSCustomLog.protocol.MI_StringA(customlog['FilePath'])
-##
-##        e = copy.deepcopy(d)
-##        t = { 'Name': 'SimpleCustomLog', 'EnableCustomLogConfiguration': True, 'CustomLogObjects': [{ 'LogName': 'LinuxSampleCustomLog1', 'FilePath': [ '/tmp/logs/*.log', '/tmp/test1.log' ] }, { 'LogName': 'LinuxSampleCustomLog2', 'FilePath': [ '/tmp/test2.log' ] } ] }
-##
-##        self.assertTrue(nxOMSCustomLog.Set_Marshall(**d) == [0],'Set('+repr(d)+') should return == [0]')
-##         
-##        m = self.make_MI(0,**t)
-##        g = nxOMSCustomLog.Get_Marshall(**e)
-##        self.assertTrue(check_values(g, m)  ==  True, 'Get('+repr(g)+' should return ==['+repr(m)+']')
-##
-##    def testSetOMSCustomLog_del(self):
-##        d = { 'Name': 'SimpleCustomLog', 'EnableCustomLogConfiguration': True, 'CustomLogObjects': None }
-##        self.assertTrue(nxOMSCustomLog.Set_Marshall(**d) == [0],'Set('+repr(d)+') should return == [0]') 
-##
-##    def testGetOMSCustomLog_default(self):
-##        d = { 'Name': 'SimpleCustomLog' }
-##        self.assertTrue(nxOMSCustomLog.Set_Marshall(**d) == [0],'Set('+repr(d)+') should return == [0]')
-##
-##        t = { 'Name': 'SimpleCustomLog', 'EnableCustomLogConfiguration': False, 'CustomLogObjects': None }
-##        m=self.make_MI(0,**t)
-##        g=nxOMSCustomLog.Get_Marshall(**d)
-##        print 'GET '+ repr(g)
-##        self.assertTrue(check_values(g, m)  ==  True, \
-##        'Get('+repr(g)+' should return ==['+repr(m)+']')
-##
-##    def testGetOMSCustomLog_del(self):
-##        d = { 'Name': 'SimpleCustomLog', 'EnableCustomLogConfiguration': True, 'CustomLogObjects': None }
-##        self.assertTrue(nxOMSCustomLog.Set_Marshall(**d) == [0],'Set('+repr(d)+') should return == [0]')
-##        t = { 'Name': 'SimpleCustomLog', 'EnableCustomLogConfiguration': True, 'CustomLogObjects': None }
-##        m=self.make_MI(0,**t)
-##        g=nxOMSCustomLog.Get_Marshall(**d)
-##        print 'GET '+ repr(g)
-##        self.assertTrue(check_values(g, m)  ==  True, \
-##        'Get('+repr(g)+' should return ==['+repr(m)+']')
+class nxOMSCustomLogTestCases(unittest2.TestCase):
+    """
+    Test Case for nxOMSCustomLog.py
+    """
+
+    original_conf_path = None
+    mock_conf_path = './ut_customlog.conf'
+
+    def setUp(self):
+        """
+        Setup test resources
+        """
+        self.original_conf_path = nxOMSCustomLog.conf_path
+        nxOMSCustomLog.conf_path = self.mock_conf_path
+        os.system('rm -rf %s' % self.mock_conf_path)
```

Comment:
Try using ruby if possible. http://ruby-doc.org/core-2.1.4/File.html#method-c-delete
This is a file, no need to recursively remove.